pvnet/data/datamodule.py

```diff
@@ -1,57 +1,138 @@
 """ Data module for pytorch lightning """
+from datetime import datetime
+from glob import glob
 
-import resource
-
+from lightning.pytorch import LightningDataModule
+from torch.utils.data import Dataset, DataLoader
 import torch
-from ocf_datapipes.batch import batch_to_tensor, stack_np_examples_into_batch
-from ocf_datapipes.training.pvnet import pvnet_datapipe
-from torch.utils.data.datapipes.iter import FileLister
 
-from pvnet.data.base import BaseDataModule
+from ocf_datapipes.batch import batch_to_tensor, stack_np_examples_into_batch, NumpyBatch
+from ocf_data_sampler.torch_datasets.pvnet_uk_regional import (
+    PVNetUKRegionalDataset
+)
+
+
+def fill_nans_in_arrays(batch):
```

Comment:
This was put into the data-sampler in openclimatefix/ocf-data-sampler#49 so could be removed now